notes/installation-guide.md

```diff
@@ -3,19 +3,19 @@ Installation instructions
 
 Note: the full story on installing Agda libraries can be found at [readthedocs](http://agda.readthedocs.io/en/latest/tools/package-system.html).
 
-Use version v1.7 of the standard library with Agda 2.6.2.
+Use version v1.7.1 of the standard library with Agda 2.6.2.1 or 2.6.2.
```

Comment:
While you are at it, the lastest release of Agda is 2.6.2.2.

Reply:
Does https://wiki.portal.chalmers.se/agda/pmwiki.php?n=Libraries.StandardLibrary also need to be updated to mention 2.6.2.2?

Reply:
Indeed.  Just did that.